### PR TITLE
CIV-10567 CIV-10978 Task for NotSuitableSDO events

### PR DESCRIPTION
**Pull request by ShwetaTandel-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-10567

https://tools.hmcts.net/jira/browse/CIV-10978

### Change description ###
WA Task: create 'Request for transfer online case' task name
(Re)Create SDO task when judge decided not to draw order and transfer the case

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by Azam-Hmcts — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java [hunk 1]
```diff
@@ -31,12 +31,13 @@ public static void initialization() {
         CURRENT_DMN_DECISION_TABLE = DmnDecisionTable.WA_TASK_CONFIGURATION_CIVIL_DAMAGES;
     }
 
+
```
Comment (hmcts-version1-ignacio) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java:34]: Extra blank line?

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java [hunk 2]
```diff
@@ -614,6 +615,40 @@ void when_taskId_review_case_flags_defendant_then_return_expected_decision() {
         )));
     }
 
+    @Test
+    void when_taskId_then_return_routine_work_desc() {
+
+        Map<String, Object> caseData = new HashMap<>();
+        caseData.put("applicant1", Map.of(
+            "partyName", "Firstname LastName"
+
```
Comment (hmcts-version1-ignacio) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java:624]: Extra blank line?

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java [hunk 3]
```diff
@@ -614,6 +615,40 @@ void when_taskId_review_case_flags_defendant_then_return_expected_decision() {
...
+        ));
+        caseData.put("applicant2", Map.of(
+            "partyName", "Firstname LastName"
+
```
Comment (hmcts-version1-ignacio) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaConfigurationTest.java:628]: Extra blank line?

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaInitiationTest.java [hunk 4]
```diff
@@ -65,10 +65,179 @@ void given_input_should_return_dj_outcome_dmn() {
...
+        List<Map<String, Object>> workTypeResultList = dmnDecisionTableResult.getResultList();
+
+        assertThat(workTypeResultList.size(), is(2));
+        assertThat(workTypeResultList
+                       .get(0).get("taskId"), is("transferOnlineCase"));
+        assertThat(workTypeResultList.get(0).get("processCategories"), is("routineTransfer"));
+        assertThat(workTypeResultList
+                       .get(1).get("taskId"), is("SmallClaimsTrackDirections"));
+        assertThat(workTypeResultList.get(1).get("processCategories"), is("standardDirectionsOrder"));
+
```
Comment (hmcts-version1-ignacio) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaInitiationTest.java:95]: Extra blank line?

File: src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaCancellationTest.java [hunk 5]
```diff
@@ -109,12 +149,14 @@ public static Stream<Arguments> scenarioTakesCaseOfflineEventCaseDismissedSystem
         );
     }
 
+
```
Comment (hmcts-version1-ignacio) [inline src/test/java/uk/gov/hmcts/civil/taskconfiguration/dmn/CamundaTaskWaCancellationTest.java:152]: Extra blank line?

**Review by hmcts-version1-ignacio — COMMENTED** [review]
Left some comments (blank lines)

**Review by hmcts-version1-ignacio — APPROVED** [review]

**Pull request merged**